plugins/modules/keycloak_client.py

```diff
@@ -249,7 +249,7 @@
             - Type of client.
             - At creation only, default value will be V(openid-connect) if O(protocol) is omitted.
```

Comment:
```suggestion
            - At creation only, default value will be V(openid-connect) if O(protocol) is omitted.
            - The V(docker-v2) value was added in community.general 8.6.0.
```
Can you add similar texts to the other options where you add new values? Thanks.